src/notebook/plugin.ts

```diff
@@ -179,6 +194,42 @@ class NotebookFileHandler extends AbstractFileHandler {
     return ['.ipynb']
   }
 
+  executeSelectedCell(): void {
+    for (let w of this.widgets) {
+      if (w.hasClass('jp-mod-focus')) {
```

Comment:
The notebook shouldn't depend on the magic value of the focus class that is defined in the application plugin. I think it's probably okay to not even do the focus check.